phoenix5-spark3/src/main/java/org/apache/phoenix/spark/sql/connector/writer/PhoenixDataSourceWriteOptions.java

```diff
@@ -28,20 +28,20 @@
 class PhoenixDataSourceWriteOptions implements Serializable {
 
     private final String tableName;
-    private final String zkUrl;
+    private final String jdbcUrl;
     private final String tenantId;
     private final String scn;
     private final StructType schema;
     private final boolean skipNormalizingIdentifier;
     private final Properties overriddenProps;
 
-    private PhoenixDataSourceWriteOptions(String tableName, String zkUrl, String scn,
+    private PhoenixDataSourceWriteOptions(String tableName, String jdbcUrl, String scn,
                                           String tenantId, StructType schema, boolean skipNormalizingIdentifier,
                                           Properties overriddenProps) {
         if (tableName == null) {
             throw new NullPointerException();
         }
-        if (zkUrl == null) {
+        if (jdbcUrl == null) {
```

Comment:
Why don't you throw IllegalArgumentException if the schema or the overriddenPropsos null?
Like it is done int the phoenix5-spark connector

Reply:
Corrected.